source/renderer/app/components/wallet/settings/WalletRecoveryPhraseVerificationWidget.scss

```diff
@@ -105,4 +105,5 @@
 
 .paperWalletTooltip {
   line-height: 22px;
+  width: 40em;
```

Comment:
@DominikGuzei can you please rather use "px" here? We are not using "em" anywhere else...

Reply:
done 👍